Update lexicon GitHub Actions fails

The update lexicons actions currently fails with the following error:

Update lexicons · tsky-dev/tsky@286616e - https://github.com/tsky-dev/tsky/actions/runs/16898468632/job/47872963658#step:5:43

```shell
Run pnpm install --frozen-lockfile
Scope: all 7 workspace projects
Lockfile is up to date, resolution step is skipped
Progress: resolved 1, reused 0, downloaded 0, added 0
Packages: +820
++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
Progress: resolved 820, reused 820, downloaded 0, added 576
Progress: resolved 820, reused 820, downloaded 0, added 820, done
 WARN  Failed to create bin at /home/runner/work/tsky/tsky/node_modules/.pnpm/node_modules/.bin/pds. ENOENT: no such file or directory, open '/home/runner/work/tsky/tsky/node_modules/.pnpm/node_modules/@atproto/pds/dist/bin.js'
 WARN  Failed to create bin at /home/runner/work/tsky/tsky/packages/internal/dev-env/node_modules/.bin/pds. ENOENT: no such file or directory, open '/home/runner/work/tsky/tsky/node_modules/.pnpm/@atproto+pds@0.4.94/node_modules/@atproto/pds/dist/bin.js'
 WARN  Failed to create bin at /home/runner/work/tsky/tsky/packages/lexicons/node_modules/.bin/lex-cli. ENOENT: no such file or directory, open '/home/runner/work/tsky/tsky/packages/lex-cli/dist/index.js'

devDependencies:
+ @biomejs/biome 1.9.4
+ nano-staged 0.8.0
+ rimraf 6.0.1

 WARN  Failed to create bin at /home/runner/work/tsky/tsky/packages/internal/dev-env/node_modules/.bin/pds. ENOENT: no such file or directory, open '/home/runner/work/tsky/tsky/packages/internal/dev-env/node_modules/@atproto/pds/dist/bin.js'
. prepare$ pnpm --filter !@tsky/docs build
packages/lex-cli prepare$ pnpm run clean && pnpm run build
packages/lex-cli prepare: > @tsky/lex-cli@0.0.1 clean /home/runner/work/tsky/tsky/packages/lex-cli
packages/lex-cli prepare: > rimraf dist
. prepare: Scope: 5 of 7 workspace projects
. prepare: packages/internal/dev-env build$ tsc
. prepare: packages/lex-cli build$ tsc
packages/lex-cli prepare: > @tsky/lex-cli@0.0.1 build /home/runner/work/tsky/tsky/packages/lex-cli
packages/lex-cli prepare: > tsc
. prepare: packages/internal/dev-env build: Done
. prepare: packages/lex-cli build: Done
. prepare: packages/lexicons build$ tsx ./scripts/generate-types.ts && tsc
. prepare: packages/lexicons build: Getting latest lexicon commit...
packages/lex-cli prepare: Done
. prepare: packages/lexicons build: Downloading lexicons from atproto...
. prepare: packages/lexicons build: Building lex-cli...
. prepare: packages/lexicons build: > @tsky/lex-cli@0.0.1 build /home/runner/work/tsky/tsky/packages/lex-cli
. prepare: packages/lexicons build: > tsc
. prepare: packages/lexicons build: Running lex-cli to generate types...
. prepare: packages/lexicons build: error: failed to read /home/runner/work/tsky/tsky/packages/lexicons/lexicons/app/bsky/notification/putPreferencesV2.json
. prepare: packages/lexicons build:   caused by: Invalid document schema
. prepare: packages/lexicons build: Error: Error: Command failed: node /home/runner/work/tsky/tsky/packages/lex-cli/dist/index.js generate-types /home/runner/work/tsky/tsky/packages/lexicons/lexicons/app/bsky/actor/defs.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/app/bsky/actor/getPreferences.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/app/bsky/actor/getProfile.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/app/bsky/actor/getProfiles.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/app/bsky/actor/getSuggestions.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/app/bsky/actor/profile.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/app/bsky/actor/putPreferences.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/app/bsky/actor/searchActors.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/app/bsky/actor/searchActorsTypeahead.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/app/bsky/actor/status.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/app/bsky/embed/defs.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/app/bsky/embed/external.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/app/bsky/embed/images.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/app/bsky/embed/record.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/app/bsky/embed/recordWithMedia.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/app/bsky/embed/video.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/app/bsky/feed/defs.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/app/bsky/feed/describeFeedGenerator.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/app/bsky/feed/generator.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/app/bsky/feed/getActorFeeds.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/app/bsky/feed/getActorLikes.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/app/bsky/feed/getAuthorFeed.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/app/bsky/feed/getFeed.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/app/bsky/feed/getFeedGenerator.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/app/bsky/feed/getFeedGenerators.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/app/bsky/feed/getFeedSkeleton.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/app/bsky/feed/getLikes.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/app/bsky/feed/getListFeed.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/app/bsky/feed/getPostThread.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/app/bsky/feed/getPosts.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/app/bsky/feed/getQuotes.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/app/bsky/feed/getRepostedBy.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/app/bsky/feed/getSuggestedFeeds.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/app/bsky/feed/getTimeline.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/app/bsky/feed/like.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/app/bsky/feed/post.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/app/bsky/feed/postgate.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/app/bsky/feed/repost.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/app/bsky/feed/searchPosts.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/app/bsky/feed/sendInteractions.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/app/bsky/feed/threadgate.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/app/bsky/graph/block.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/app/bsky/graph/defs.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/app/bsky/graph/follow.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/app/bsky/graph/getActorStarterPacks.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/app/bsky/graph/getBlocks.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/app/bsky/graph/getFollowers.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/app/bsky/graph/getFollows.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/app/bsky/graph/getKnownFollowers.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/app/bsky/graph/getList.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/app/bsky/graph/getListBlocks.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/app/bsky/graph/getListMutes.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/app/bsky/graph/getLists.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/app/bsky/graph/getListsWithMembership.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/app/bsky/graph/getMutes.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/app/bsky/graph/getRelationships.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/app/bsky/graph/getStarterPack.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/app/bsky/graph/getStarterPacks.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/app/bsky/graph/getStarterPacksWithMembership.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/app/bsky/graph/getSuggestedFollowsByActor.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/app/bsky/graph/list.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/app/bsky/graph/listblock.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/app/bsky/graph/listitem.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/app/bsky/graph/muteActor.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/app/bsky/graph/muteActorList.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/app/bsky/graph/muteThread.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/app/bsky/graph/searchStarterPacks.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/app/bsky/graph/starterpack.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/app/bsky/graph/unmuteActor.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/app/bsky/graph/unmuteActorList.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/app/bsky/graph/unmuteThread.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/app/bsky/graph/verification.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/app/bsky/labeler/defs.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/app/bsky/labeler/getServices.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/app/bsky/labeler/service.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/app/bsky/notification/declaration.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/app/bsky/notification/defs.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/app/bsky/notification/getPreferences.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/app/bsky/notification/getUnreadCount.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/app/bsky/notification/listActivitySubscriptions.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/app/bsky/notification/listNotifications.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/app/bsky/notification/putActivitySubscription.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/app/bsky/notification/putPreferences.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/app/bsky/notification/putPreferencesV2.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/app/bsky/notification/registerPush.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/app/bsky/notification/unregisterPush.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/app/bsky/notification/updateSeen.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/app/bsky/richtext/facet.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/app/bsky/unspecced/defs.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/app/bsky/unspecced/getAgeAssuranceState.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/app/bsky/unspecced/getConfig.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/app/bsky/unspecced/getPopularFeedGenerators.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/app/bsky/unspecced/getPostThreadOtherV2.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/app/bsky/unspecced/getPostThreadV2.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/app/bsky/unspecced/getSuggestedFeeds.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/app/bsky/unspecced/getSuggestedFeedsSkeleton.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/app/bsky/unspecced/getSuggestedStarterPacks.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/app/bsky/unspecced/getSuggestedStarterPacksSkeleton.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/app/bsky/unspecced/getSuggestedUsers.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/app/bsky/unspecced/getSuggestedUsersSkeleton.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/app/bsky/unspecced/getSuggestionsSkeleton.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/app/bsky/unspecced/getTaggedSuggestions.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/app/bsky/unspecced/getTrendingTopics.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/app/bsky/unspecced/getTrends.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/app/bsky/unspecced/getTrendsSkeleton.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/app/bsky/unspecced/initAgeAssurance.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/app/bsky/unspecced/searchActorsSkeleton.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/app/bsky/unspecced/searchPostsSkeleton.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/app/bsky/unspecced/searchStarterPacksSkeleton.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/app/bsky/video/defs.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/app/bsky/video/getJobStatus.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/app/bsky/video/getUploadLimits.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/app/bsky/video/uploadVideo.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/chat/bsky/actor/declaration.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/chat/bsky/actor/defs.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/chat/bsky/actor/deleteAccount.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/chat/bsky/actor/exportAccountData.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/chat/bsky/convo/acceptConvo.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/chat/bsky/convo/addReaction.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/chat/bsky/convo/defs.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/chat/bsky/convo/deleteMessageForSelf.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/chat/bsky/convo/getConvo.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/chat/bsky/convo/getConvoAvailability.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/chat/bsky/convo/getConvoForMembers.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/chat/bsky/convo/getLog.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/chat/bsky/convo/getMessages.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/chat/bsky/convo/leaveConvo.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/chat/bsky/convo/listConvos.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/chat/bsky/convo/muteConvo.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/chat/bsky/convo/removeReaction.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/chat/bsky/convo/sendMessage.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/chat/bsky/convo/sendMessageBatch.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/chat/bsky/convo/unmuteConvo.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/chat/bsky/convo/updateAllRead.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/chat/bsky/convo/updateRead.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/chat/bsky/moderation/getActorMetadata.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/chat/bsky/moderation/getMessageContext.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/chat/bsky/moderation/updateActorAccess.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/com/atproto/admin/defs.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/com/atproto/admin/deleteAccount.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/com/atproto/admin/disableAccountInvites.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/com/atproto/admin/disableInviteCodes.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/com/atproto/admin/enableAccountInvites.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/com/atproto/admin/getAccountInfo.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/com/atproto/admin/getAccountInfos.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/com/atproto/admin/getInviteCodes.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/com/atproto/admin/getSubjectStatus.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/com/atproto/admin/searchAccounts.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/com/atproto/admin/sendEmail.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/com/atproto/admin/updateAccountEmail.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/com/atproto/admin/updateAccountHandle.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/com/atproto/admin/updateAccountPassword.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/com/atproto/admin/updateAccountSigningKey.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/com/atproto/admin/updateSubjectStatus.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/com/atproto/identity/defs.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/com/atproto/identity/getRecommendedDidCredentials.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/com/atproto/identity/refreshIdentity.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/com/atproto/identity/requestPlcOperationSignature.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/com/atproto/identity/resolveDid.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/com/atproto/identity/resolveHandle.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/com/atproto/identity/resolveIdentity.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/com/atproto/identity/signPlcOperation.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/com/atproto/identity/submitPlcOperation.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/com/atproto/identity/updateHandle.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/com/atproto/label/defs.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/com/atproto/label/queryLabels.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/com/atproto/label/subscribeLabels.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/com/atproto/lexicon/schema.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/com/atproto/moderation/createReport.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/com/atproto/moderation/defs.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/com/atproto/repo/applyWrites.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/com/atproto/repo/createRecord.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/com/atproto/repo/defs.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/com/atproto/repo/deleteRecord.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/com/atproto/repo/describeRepo.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/com/atproto/repo/getRecord.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/com/atproto/repo/importRepo.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/com/atproto/repo/listMissingBlobs.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/com/atproto/repo/listRecords.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/com/atproto/repo/putRecord.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/com/atproto/repo/strongRef.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/com/atproto/repo/uploadBlob.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/com/atproto/server/activateAccount.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/com/atproto/server/checkAccountStatus.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/com/atproto/server/confirmEmail.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/com/atproto/server/createAccount.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/com/atproto/server/createAppPassword.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/com/atproto/server/createInviteCode.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/com/atproto/server/createInviteCodes.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/com/atproto/server/createSession.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/com/atproto/server/deactivateAccount.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/com/atproto/server/defs.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/com/atproto/server/deleteAccount.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/com/atproto/server/deleteSession.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/com/atproto/server/describeServer.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/com/atproto/server/getAccountInviteCodes.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/com/atproto/server/getServiceAuth.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/com/atproto/server/getSession.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/com/atproto/server/listAppPasswords.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/com/atproto/server/refreshSession.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/com/atproto/server/requestAccountDelete.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/com/atproto/server/requestEmailConfirmation.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/com/atproto/server/requestEmailUpdate.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/com/atproto/server/requestPasswordReset.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/com/atproto/server/reserveSigningKey.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/com/atproto/server/resetPassword.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/com/atproto/server/revokeAppPassword.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/com/atproto/server/updateEmail.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/com/atproto/sync/defs.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/com/atproto/sync/getBlob.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/com/atproto/sync/getBlocks.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/com/atproto/sync/getCheckout.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/com/atproto/sync/getHead.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/com/atproto/sync/getHostStatus.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/com/atproto/sync/getLatestCommit.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/com/atproto/sync/getRecord.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/com/atproto/sync/getRepo.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/com/atproto/sync/getRepoStatus.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/com/atproto/sync/listBlobs.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/com/atproto/sync/listHosts.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/com/atproto/sync/listRepos.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/com/atproto/sync/listReposByCollection.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/com/atproto/sync/notifyOfUpdate.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/com/atproto/sync/requestCrawl.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/com/atproto/sync/subscribeRepos.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/com/atproto/temp/addReservedHandle.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/com/atproto/temp/checkHandleAvailability.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/com/atproto/temp/checkSignupQueue.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/com/atproto/temp/fetchLabels.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/com/atproto/temp/requestPhoneVerification.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/tools/ozone/communication/createTemplate.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/tools/ozone/communication/defs.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/tools/ozone/communication/deleteTemplate.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/tools/ozone/communication/listTemplates.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/tools/ozone/communication/updateTemplate.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/tools/ozone/hosting/getAccountHistory.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/tools/ozone/moderation/defs.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/tools/ozone/moderation/emitEvent.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/tools/ozone/moderation/getAccountTimeline.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/tools/ozone/moderation/getEvent.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/tools/ozone/moderation/getRecord.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/tools/ozone/moderation/getRecords.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/tools/ozone/moderation/getRepo.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/tools/ozone/moderation/getReporterStats.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/tools/ozone/moderation/getRepos.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/tools/ozone/moderation/getSubjects.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/tools/ozone/moderation/queryEvents.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/tools/ozone/moderation/queryStatuses.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/tools/ozone/moderation/searchRepos.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/tools/ozone/safelink/addRule.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/tools/ozone/safelink/defs.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/tools/ozone/safelink/queryEvents.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/tools/ozone/safelink/queryRules.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/tools/ozone/safelink/removeRule.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/tools/ozone/safelink/updateRule.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/tools/ozone/server/getConfig.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/tools/ozone/set/addValues.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/tools/ozone/set/defs.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/tools/ozone/set/deleteSet.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/tools/ozone/set/deleteValues.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/tools/ozone/set/getValues.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/tools/ozone/set/querySets.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/tools/ozone/set/upsertSet.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/tools/ozone/setting/defs.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/tools/ozone/setting/listOptions.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/tools/ozone/setting/removeOptions.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/tools/ozone/setting/upsertOption.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/tools/ozone/signature/defs.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/tools/ozone/signature/findCorrelation.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/tools/ozone/signature/findRelatedAccounts.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/tools/ozone/signature/searchAccounts.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/tools/ozone/team/addMember.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/tools/ozone/team/defs.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/tools/ozone/team/deleteMember.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/tools/ozone/team/listMembers.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/tools/ozone/team/updateMember.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/tools/ozone/verification/defs.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/tools/ozone/verification/grantVerifications.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/tools/ozone/verification/listVerifications.json /home/runner/work/tsky/tsky/packages/lexicons/lexicons/tools/ozone/verification/revokeVerifications.json -o /home/runner/work/tsky/tsky/packages/lexicons/src/lib/lexicons.ts --description "Contains type declarations for Bluesky lexicons" --metadata "{\"sha\":\"c370d933b76b4e15b83a82b40d1b6a32bd54add6\",\"version\":\"main\",\"sourceUrl\":\"[https://github.com/bluesky-social/atproto/tree/c370d933b76b4e15b83a82b40d1b6a32bd54add6/lexicons\](https://github.com/bluesky-social/atproto/tree/c370d933b76b4e15b83a82b40d1b6a32bd54add6/lexicons/)"}"
. prepare: packages/lexicons build:     at genericNodeError (node:internal/errors:984:15)
. prepare: packages/lexicons build:     at wrappedFn (node:internal/errors:538:14)
. prepare: packages/lexicons build:     at checkExecSyncError (node:child_process:891:11)
. prepare: packages/lexicons build:     at execSync (node:child_process:963:15)
. prepare: packages/lexicons build:     at main (/home/runner/work/tsky/tsky/packages/lexicons/scripts/generate-types.ts:106:5) {
. prepare: packages/lexicons build:   status: 1,
. prepare: packages/lexicons build:   signal: null,
. prepare: packages/lexicons build:   output: [ null, null, null ],
. prepare: packages/lexicons build:   pid: 2334,
. prepare: packages/lexicons build:   stdout: null,
. prepare: packages/lexicons build:   stderr: null
. prepare: packages/lexicons build: }
. prepare: packages/lexicons build: Failed
. prepare: /home/runner/work/tsky/tsky/packages/lexicons:
. prepare:  ERR_PNPM_RECURSIVE_RUN_FIRST_FAIL  @tsky/lexicons@0.0.1 build: `tsx ./scripts/generate-types.ts && tsc`
. prepare: Exit status 1
. prepare: Failed
 ELIFECYCLE  Command failed with exit code 1.
Error: Process completed with exit code 1.
```

There seems a schema error in `putPreferencesV2.json` but I'm not sure what part has an issue.

```shell
...
. prepare: packages/lexicons build: Running lex-cli to generate types...
. prepare: packages/lexicons build: error: failed to read /home/runner/work/tsky/tsky/packages/lexicons/lexicons/app/bsky/notification/putPreferencesV2.json
. prepare: packages/lexicons build:   caused by: Invalid document schema
...
```

I can see the same file on local as the official atproto repo:

atproto/lexicons/app/bsky/notification/putPreferencesV2.json at c370d933b76b4e15b83a82b40d1b6a32bd54add6 · bluesky-social/atproto - https://github.com/bluesky-social/atproto/blob/c370d933b76b4e15b83a82b40d1b6a32bd54add6/lexicons/app/bsky/notification/putPreferencesV2.json

You can reproduce the same issue on local with `pnpm --filter @tsky/lexicons prepare`.

--- a/packages/client/README.md
+++ b/packages/client/README.md
@@ -10,7 +10,7 @@
 
 ## ⚠️ tsky is still in development and is not ready for production use
 
-tsky is still in active development and is not ready for production use. If you want to contribute to the project, please read the [CONTRIBUTING.md](CONTRIBUTING.md) file or join our [Discord Server](https://discord.gg/KPD7XPUZn3).
+tsky is still in active development and is not ready for production use. If you want to contribute to the project, please read the [CONTRIBUTING.md](../../CONTRIBUTING.md) file or join our [Discord Server](https://discord.gg/KPD7XPUZn3).
 
 tsky is a lightweight, fast, universal and typed Bluesky API wrapper for Apps & Bots. It's designed to be easy to use, lightweight and straightforward to use. It's built with TypeScript and has full type support.
 
@@ -77,7 +77,7 @@ const user_profile = await agent.user.profile();
 
 ## Contributing
 
-If you want to contribute to this project, please read the [CONTRIBUTING.md](CONTRIBUTING.md) file.
+If you want to contribute to this project, please read the [CONTRIBUTING.md](../../CONTRIBUTING.md) file.
 
 ## License
 

--- a/packages/lex-cli/src/generator/schema.ts
+++ b/packages/lex-cli/src/generator/schema.ts
@@ -368,9 +368,25 @@ export const userTypeSchema = t.isOneOf([
 
 export type UserTypeSchema = t.InferType<typeof userTypeSchema>;
 
+/**
+ * represents a namespace identifier (NSID)
+ */
+export type Nsid = `${string}.${string}.${string}`;
+
 const NSID_RE =
-  /^[a-z](?:[a-z0-9-]{0,61}[a-z0-9])?(?:\.[a-z0-9](?:[a-z0-9-]{0,61}[a-z0-9])?)+\.[a-z]{1,63}$/i;
-const nsidType = t.cascade(t.isString(), (value) => NSID_RE.test(value));
+  /^[a-zA-Z](?:[a-zA-Z0-9-]{0,61}[a-zA-Z0-9])?(?:\.[a-zA-Z0-9](?:[a-zA-Z0-9-]{0,61}[a-zA-Z0-9])?)+\.[a-zA-Z][a-zA-Z0-9]{0,62}?$/;
+
+// #__NO_SIDE_EFFECTS__
+export const isNsid = (input: unknown): input is Nsid => {
+  return (
+    typeof input === 'string' &&
+    input.length >= 5 &&
+    input.length <= 317 &&
+    NSID_RE.test(input)
+  );
+};
+
+const nsidType = t.cascade(t.isString(), (value) => isNsid(value));
 
 export const documentSchema = t.cascade(
   t.isObject({

